testing/regression/cypress/e2e/pages/page-library/pagination.page.js

```diff
@@ -36,6 +36,7 @@ class PaginationPage {
     }
 
     get openInvestigationTable() {
+        cy.wait(1500);
         return cy.get(this.table).eq(0);
```

Comment:
Suggestion: I think you can do this:
```javascript
cy.get('this.table', { timeout: 6000 })
```

Not sure though.

Reply:
some tests failing Mike